spec/beaker/host/unix/exec_spec.rb

```diff
@@ -114,31 +112,6 @@ def to_s
           expect { instance.ssh_service_restart }.not_to raise_error
         end
       end
-
-      PlatformHelpers::DEBIANPLATFORMS.each do |platform|
-        it "calls the correct command for #{platform}" do
-          opts['platform'] = platform
-          expect(instance).to receive(:exec)
-          expect(Beaker::Command).to receive(:new).with("service ssh restart")
-          expect { instance.ssh_service_restart }.not_to raise_error
-        end
-      end
-
-      PlatformHelpers::SYSTEMVPLATFORMS.each do |platform|
```

Comment:
Would have been nice to keep this for EL6.

Reply:
@joshcooper do you still rely on it? Vox Pupuli doesn't support EL6 anymore and I suspect you won't update to beaker 6 soonish and drop EL6 from your pipelines anyways?

Reply:
We special cased it in the code. Either we should have dropped it or implemented tests. We kept it in at @joshcooper's request so I'd like to test it